app/src/main/res/layout/organizer_detail_layout.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout>
+
+    <data>
+        <import type="android.view.View" />
+        <import type="org.fossasia.openevent.app.common.utils.core.DateUtils" />
+
+        <variable
+            name="user"
+            type="org.fossasia.openevent.app.common.data.models.User" />
+    </data>
+
+    <android.support.v4.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical">
+
+            <ImageView
+                android:layout_width="@dimen/avatar_size"
+                android:layout_height="@dimen/avatar_size"
+                android:layout_gravity="center"
+                app:circleImageUrl="@{ user.avatarUrl }"
+                android:layout_margin="@dimen/spacing_medium"
+                app:placeholder="@{ @drawable/ic_account_circle }"
+                android:contentDescription="@string/profile_picture" />
+
+            <android.support.v7.widget.CardView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/spacing_medium" >
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:padding="@dimen/spacing_small"
+                        android:text="@string/details"
+                        android:background="#dedede"
+                        android:textAppearance="@style/TextAppearance.AppCompat.Headline"
```

Comment:
Every activity has a toolbar and can contain larger TextViews. "In certain scenarios, use the larger subheading style instead of the smaller Body style. Some of those scenarios include when information is presented as small snippets or when titles are paired with lines of Body-styled text."

Because the title has to be always 20sp - "Across form factors, text that appears in the app bar should use the Title style, Medium 20sp."